src/core/ext/transport/chttp2/server/chttp2_server.cc

```diff
@@ -273,6 +277,7 @@ class Chttp2ServerListener : public Server::ListenerInterface {
   MemoryQuotaRefPtr memory_quota_;
   ConnectionQuotaRefPtr connection_quota_;
   grpc_server_config_fetcher* config_fetcher_ = nullptr;
+  std::shared_ptr<experimental::PassiveListenerImpl> passive_listener_;
```

Comment:
Consider using `absl::variant<>` to minimize memory usage for disjoint cases where different fields are used.  Or just add a TODO for @yashykt to consider this as part of his refactoring.

Reply:
Done